packages/app-project/src/shared/components/ProjectStatistics/components/Subtitle/Subtitle.js

```diff
@@ -1,12 +1,16 @@
-import { Text } from 'grommet'
+import { ResponsiveContext, Text } from 'grommet'
 import { object, oneOfType, string } from 'prop-types'
 import React from 'react'
 
-function Subtitle ({ mode, text, ...props }) {
+function Subtitle ({ margin, mode, text, ...props }) {
   return (
-    <Text {...props}>
-      {text}
-    </Text>
+    <ResponsiveContext.Consumer>
+      {(size) => (
+        <Text {...props} margin={(size === 'small') ? { top: 'none', bottom: 'xsmall' } : margin}>
```

Comment:
Instead of changing props based on size, couldn't we change the values of `global.breakpoints.small.edgeSize` to be smaller so they're applied across the theme?

Reply:
This PR includes an update to the global small edge sizes to increase them slightly. This is the only case where it needed to vary. If we make the size smaller again, there's a cascading effect of the margins and paddings not being enough for other components.